client/src/pages/learn/back-end-development-and-apis/managing-packages-with-npm/index.md

```diff
@@ -24,4 +24,4 @@ Working on these challenges involves writing code on Replit in our starter proje
 
 Optionally, you may write your project on another platform, but it must be publicly visible for our testing.
 
-Start this project on Replit using <a href='https://replit.com/github/freeCodeCamp/boilerplate-npm'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub! If you use Replit, remember to save the link to your project somewhere safe!
+Start this project by cloning <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub!
```

Comment:
Is the below reference to Replit still required considering Codespace in GitHub? 

![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/30977545/ef78f7b9-659c-436f-984b-63a1195125e9)

Reply:
@VKSGit2017 thank you for your question, it looks like #54114 is dealing with that file